src/main/deltachat/webxdc.ts

```diff
@@ -430,6 +432,30 @@ If you think that's a bug and you need that permission, then please open an issu
       this._closeAll()
     })
 
+    ipcMain.handle(
+      'webxdc:custom:drag-file-out',
+      async (
+        event,
+        file_name: string,
+        base64_content: string,
+        icon_data_url?: string
+      ) => {
+        const path = await writeTempFileFromBase64(file_name, base64_content)
+        let icon: string | Electron.NativeImage = join(
+          __dirname,
+          '../../../images/electron-file-drag-out.png'
+        )
+        if (icon_data_url) {
+          icon = nativeImage.createFromDataURL(icon_data_url)
+        }
```

Comment:
```suggestion
        let icon: string | Electron.NativeImage = icon_data_url ? nativeImage.createFromDataURL(icon_data_url) : join(
          __dirname,
          '../../../images/electron-file-drag-out.png'
        )
```

Reply:
does not really matter here, so I would keep it